analyzers/src/SonarAnalyzer.CSharp/Metrics/CSharpExecutableLinesMetric.cs

```diff
@@ -109,6 +109,14 @@ private bool FindExecutableLines(SyntaxNode node)
                         ExecutableLineNumbers.Add(node.GetLocation().GetLineNumberToReport());
                         return true;
 
+                    case SyntaxKindEx.SwitchExpressionArm:
```

Comment:
please read this comment: https://github.com/SonarSource/sonar-dotnet/pull/4663/commits/a7d2a8624895d9acc539dc20948d5ec7bba2b5e3#r670270653